lib/interfaces/include/CCUEthernetInterface.h

```diff
@@ -3,18 +3,13 @@
 
 
 #include "hytech_msgs.pb.h"
+#include "SharedFirmwareTypes.h"
```

Comment:
In the future, please name all branches fix/ or feature/ or test/ to mark it as a relevant "type" of branch. See Ben's firmware guide (linked at the end of Module 4). Don't worry about fixing it for this branch, just a heads-up for the future.